docker-compose-test.yml

```diff
@@ -0,0 +1,12 @@
+version: '3.1'
```

Comment:
I assume this needs to be executed when running int tests - if so, it should be part of the README